src/System.Private.CoreLib/shared/System/Globalization/CultureInfo.cs

```diff
@@ -701,15 +836,15 @@ public virtual CompareInfo CompareInfo
         {
             get
             {
-                if (this.compareInfo == null)
+                if (_compareInfo == null)
                 {
                     // Since CompareInfo's don't have any overrideable properties, get the CompareInfo from
                     // the Non-Overridden CultureInfo so that we only create one CompareInfo per culture
-                    this.compareInfo = UseUserOverride
-                                        ? GetCultureInfo(this._name).CompareInfo
-                                        : new CompareInfo(this);
+                    _compareInfo = UseUserOverride
+                                    ? GetCultureInfo(_name).CompareInfo
+                                    : new CompareInfo(this);
                 }
-                return (compareInfo);
+                return (_compareInfo);
```

Comment:
Nit: Remove the superfluous braces in return statements in the remaining places in this file?